task/backend/scheduler/scheduler.go

```diff
@@ -13,15 +13,6 @@ import (
 // TODO(lh): maybe make this its own thing sometime in the future.
 type ID influxdb.ID
 
-// Checkpointer allows us to restart a service from the last time we executed.
```

Comment:
The interface is required for the scheduler. You are writing a implementation of this interface. We will have several implementations of the interface.